integration/app/test-demo/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "test-demo",
+  "version": "0.1.0",
+  "dependencies": {
+    "@types/react": "^17.0.33",
+    "@types/react-dom": "^17.0.10",
+    "@types/react-router-dom": "^5.3.2",
+    "@types/styled-components": "^5.1.15",
+    "@types/webpack-env": "^1.16.3",
+    "@typescript-eslint/eslint-plugin": "^5.2.0",
+    "@typescript-eslint/parser": "^5.2.0",
+    "amazon-chime-sdk-component-library-react": "^2.11.1",
+    "amazon-chime-sdk-js": "^2.20.0",
+    "aws-sdk": "^2.1015.0",
+    "concurrently": "^6.3.0",
+    "html-webpack-plugin": "^5.5.0",
+    "react": "^17.0.2",
+    "react-dev-utils": "^11.0.4",
+    "react-dom": "^17.0.2",
+    "react-router-dom": "^5.3.0",
+    "source-map-loader": "^3.0.0",
+    "styled-components": "^5.3.3",
+    "styled-system": "^5.1.5",
+    "ts-loader": "^9.2.6",
+    "typescript": "^4.4.4",
+    "uuid": "^8.3.2",
+    "webpack-cli": "^4.9.1",
+    "webpack-dev-server": "^4.3.1"
+  },
+  "scripts": {
+    "lint": "eslint src --ext .ts,.tsx --fix",
```

Comment:
where is the lint config file?

Reply:
It's not supposed to have this line. Deleted.